src/cct.c

```diff
@@ -173,6 +173,11 @@ int main(int argc, char **argv) {
     direction = opt_given (o, "I")? -1: 1;
     verbose   = opt_given (o, "v");
 
+    if (opt_given (o, "version")) {
+        fprintf (stdout, "%s: %s\n", o->progname, pj_get_release ());
```

Comment:
Same as above

Reply:
Yes, I agree, but cf. PR #775 to see why I didn't do it that way. I suggest, for now, to
1. merge this PR (which is tidy in the sense of implementing --version in an identical way to cs2cs and proj),
2. Then wait for #775 to get merged,
3. And finally update cct & gie in a new (small) PR.